components/calendar/CalToggleButton.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <div class="control-row is-hidden-mobile">
```

Comment:
how does this look on a tablet? we might want to think about hiding at a slightly larger size

Reply:
I changed to `is-hidden-touch` so it's not visible on tablets under a certain size, in addition to phones.